seirmo/apps/seir_simulation_app.py

```diff
@@ -68,26 +69,34 @@
  infecting rate of the disease, many models are used to characterise the
  disease, such as the SEIR model and the branching processes model.  
   
-SEIR model is a deterministic model and has been used to model infectious
- disease.  """
+SEIR model is a deterministic model. A deterministic model means that the simulation will always 
+be the same as long as the parameters are the same. This SEIR model has been used to model 
+infectious disease.  """
 
 # Model description
 description = """
 The SEIR model is a model of Ordinary Differential Equations (ODEs). 
 It assigns a population into 4 groups, which are Susceptible(S), Exposed(E), Infectious(I) 
 and Recovered(R). In this model, individuals will transit from being Susceptible to Exposed, from 
-Exposed to Infectious and from Infectious to Recovered. To control the rate at which the individuals
-transit from one state to the other, some parameters are imposed in the model.  
+Exposed to Infectious and from Infectious to Recovered.  
   
-The model is characterised by few constants, which include the *reproduction number*, the 
-*incubation period* and the *infection period*. The *reproduction number* measures the number of 
-infected cases originating from primary infections, the *incubation period* defines the average 
-period of time for exposed individuals to become infectious, and the *infection period* is the 
-average period of time for infected patients to recover from the disease.  
+The transition from one group to the other is controlled or described by different rates. The 
+*infection rate* indicates the speed at which an individual from the S gets exposed and enters the 
+group E. The *incubation rate* explains how fast an individual who got exposed becomes infected. For 
+example, if the incubation rate of the disease is higher, then individuals get infected quickly once 
+it is exposed. Finally, the rate where infected individuals get recovered is described by the *recovery 
+rate*. 
   
-The system of ODEs is solved to retrieve the number of inviduals in each S, E, I and R group. The
-incidence number is then inferred from the solution. To solve the system of ODEs, the initial value of
-each group is required. Different initial values will give different solutions.  
+The bar graph shows the daily incidence number of an infectious disease. The blue bars show actual 
```

Comment:
Can you split this up and give it a heading "Simulation and real data" or something like that?

Reply:
Just to make clear that we now transition of describing the model to the actual app.